enterprise/management/commands/seed_enterprise_devstack_data.py

```diff
@@ -242,10 +242,10 @@ def handle(self, *args, **options):
         LOGGER.info('\nCreating a new enterprise...')
         enterprise = self._create_enterprise(enterprise_users=enterprise_users)
         LOGGER.info(
-            '\nSuccessfully created a new enterprise with the following data:' +
-            '\n|    Enterprise Customer: %s (%s)' +
-            '\n|    Enterprise Catalog: %s (%s)' +
-            '\n|    Enterprise Users (%d): %s',
+            '''\nSuccessfully created a new enterprise with the following data:
+            \n|    Enterprise Customer: %s (%s)
+            \n|    Enterprise Catalog: %s (%s)
+            \n|    Enterprise Users (%i): %s''',
```

Comment:
To get the same string as the original, you probably need to pass this into `textwrap.dedent()`.